src/test/rustdoc/strip-block-doc-comments-stars.docblock.html

```diff
@@ -1,2 +1,6 @@
-<div class="docblock"><p>a</p>
+<div class="docblock"><ul>
+<li>
```

Comment:
> The goal of this test is to ensure that it won't be generated as a list

It’s generating a list, and it really looks like it shouldn’t.

Reply:
Oh indeed. I only saw the code block. Good catch!